includes/components/class-attribute.php

```diff
@@ -307,6 +310,70 @@ public function register_models() {
 		}
 	}
 
+	/**
+	 * Validates model attributes.
+	 *
+	 * @param array  $errors Form errors.
+	 * @param object $form Form object.
+	 * @return array
+	 */
+	public function validate_model_attributes( $errors, $form ) {
+		if ( empty( $errors ) ) {
```

Comment:
If it's the decisive condition for all the code in the function, please use a guard clause instead, e.g. if not empty errors, return errors..